pom.xml

```diff
@@ -43,6 +43,7 @@
         <air.java.version>1.8.0-161</air.java.version>
         <air.maven.version>3.3.9</air.maven.version>
 
+        <dep.alluxio.version>2.0.1</dep.alluxio.version>
```

Comment:
Inline this version since it is only used in one place. We only add properties for versions when there are multiple dependencies that all need the same version.

Reply:
done